### PR TITLE
[#106] chore: update user menu logout colour

### LINKED ISSUE #106: [TASK]: Update user dropdown - logout to red
### Task Summary

- Just update the colour of the logout button text to red

### Acceptance criteria

- [x] - The logout button that is apart of the user menu in the header is red.

### Any Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Jo9871**
---
title: [#106] | update user menu logout colour
---

Discord Username: @Jo9871 

## What type of PR is this? (select all that apply)

- 🍕 Feature
- 🐛 Bug Fix
- 🚧 Breaking Change
- [x] 🧑‍💻 Code Refactor
- 📝 Documentation Update

## Description

Updated the colour of the logout button text and focus background to red

## Related Tickets & Documents

- Related Issue #106 
- Closes #106 

## Added/updated tests?

- 👍 yes
- [x] 🙅 no, because they aren't needed
- 🙋 no, because I need help

### CODE REVIEW

File: src/components/user-dropdown.tsx [hunk 1]
```diff
@@ -1,7 +1,10 @@
 "use client";
 
 import { signOut, useSession } from "next-auth/react";
-import Spinner from "./ui/spinner";
+import Image from "next/image";
+import Link from "next/link";
+import { Icons } from "./icons";
+import { Button } from "./ui/button";
```
Comment (Lukortech) [inline src/components/user-dropdown.tsx:7]: Could you please import the stuff in one line?

**Review by Lukortech — APPROVED** [review]
Minor importing error, nothing that can't be improved easily later on.

File: src/components/user-dropdown.tsx (hunk 1 above)
Comment (Jo9871) [reply src/components/user-dropdown.tsx:7]: I'm not sure, this is how it was before I was messing with the file, plus they are all from differnt component files anyway.

Comment (Lukortech) [reply src/components/user-dropdown.tsx:7]: It's just the scouts code rule to leave code better than it was, nothing too importand but a "nice touch".

Generally speaking 
```jsx
import { 
  Button, 
  Icon, 
  AnythingINeed 
} from "./ui/button";
```
is more readable than
```jsx
import { Button } from "./ui/button";
import { Icon} from "./ui/button";
import { AnythingINeed } from "./ui/button";
```

Comment (Jo9871) [reply src/components/user-dropdown.tsx:7]: The icon isn't coming from `./ui/button`, its is coming from it's coming from a totally separate file in `./icons`

Comment (webdevcody) [reply src/components/user-dropdown.tsx:7]: lgtm, these all come from separate files

**Pull request merged**